src/Symfony/Bundle/WebServerBundle/Command/ServerRunCommand.php

```diff
@@ -129,7 +134,13 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         try {
             $server = new WebServer();
-            $config = new WebServerConfig($documentRoot, $env, $input->getArgument('addressport'), $input->getOption('router'));
+            $config = new WebServerConfig(
+                $documentRoot,
+                $env,
+                $input->getArgument('addressport'),
+                $input->getOption('router'),
+                $input->getOption('executable')
```

Comment:
I believe `executable` and `binary` in this scenario are synonyms. Bash calls it `command`, in my mind it makes a bit more sense (even though the command will be the whole string that the Process will run).